server.py

```diff
@@ -4,7 +4,7 @@
 
 
 class Handler(http.server.SimpleHTTPRequestHandler):
-    phrase = b'Hello everyone'
+    phrase = b'Hola mundo'
```

Comment:
Should this have an accent?
```suggestion
    phrase = b'Hóla mundo'
```